sympy/stats/stochastic_process_types.py

```diff
@@ -179,7 +195,7 @@ def joint_distribution(self, *args):
                 raise ValueError("Expected a RandomIndexedSymbol or "
                                 "key not  %s"%(type(arg)))
 
-        if args[0].pspace.distribution == None: # checks if there is any distribution available
+        if not hasattr(args[0].pspace.process, 'distribution'): # checks if there is any distribution available
```

Comment:
Why this change?

Reply:
By default `args[0].pspace.process.distribution` is `None`,
https://github.com/sympy/sympy/blob/22770dabc64182e2e16b0b91ba876d36f57399bc/sympy/stats/stochastic_process.py#L45-L47

Reply:
Initially, distribution was not passed to pspace while creating CTSP instance, but I fixed it in the PR too. I will remove this change
https://github.com/sympy/sympy/blob/e31c4cdedb5080325b3fd04f4b4826d9dab65b26/sympy/stats/stochastic_process_types.py#L228-L229